### ISSUE TITLE
Recursive folders 

### ISSUE THREAD

**Issue by avatsaev** [leny/atom-tree-ignore#32]
Hiding files in a folder recursively doesn't work

example: 

`./src/app/**/*.js`

**Comment by jbgarr** [comment]
Yeah, hiding recursive/nested folders would be extremely helpful. Any updates on this? Is this project still being maintained?

**Comment by leny** [comment]
@jbgarr still maintained but have a lot of things keeping me busy right now (recently became dad, and school year starting, needing to prepare courses and stuffs).

Will try to take a look when I gat time, but if someone want to dig onto it and submit a Pull Request, I won't be sad 😉
